<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeEdit</class>
 <widget class="QDialog" name="CodeEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget_left">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>0</y>
     <width>1166</width>
     <height>768</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 248, 255);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_17">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>483</x>
       <y>3</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>RFID/EKS 编码写入</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>961</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    font: 14pt &quot;黑体&quot;;
    margin-top: 1ex;
    border-width:0.5px;  
    border-style:solid;
    border-color: rgb(51, 153, 255);  
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left:15px;      
    margin-left: 5px;
    margin-right: 5px;
    padding:1px;
}</string>
     </property>
     <property name="title">
      <string>RFID写入</string>
     </property>
     <widget class="QLabel" name="label_rfidReturn">
      <property name="geometry">
       <rect>
        <x>815</x>
        <y>83</y>
        <width>131</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(250, 0, 0);</string>
      </property>
      <property name="text">
       <string>写入成功!</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_RFID">
      <property name="geometry">
       <rect>
        <x>116</x>
        <y>83</y>
        <width>231</width>
        <height>35</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Arial&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_RFIDWrite">
      <property name="geometry">
       <rect>
        <x>425</x>
        <y>83</y>
        <width>80</width>
        <height>35</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style: solid;
background-color: rgb(51,153,255);</string>
      </property>
      <property name="text">
       <string>写入</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_RFIDRead">
      <property name="geometry">
       <rect>
        <x>545</x>
        <y>83</y>
        <width>80</width>
        <height>35</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style: solid;
background-color: rgb(51,153,255);</string>
      </property>
      <property name="text">
       <string>读取</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_RFIDClear">
      <property name="geometry">
       <rect>
        <x>665</x>
        <y>83</y>
        <width>80</width>
        <height>35</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style: solid;
background-color: rgb(51,153,255);</string>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_RFIDState">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>85</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/35_all/69.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>350</y>
       <width>961</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    font: 14pt &quot;黑体&quot;;
    margin-top: 1ex;
    border-width:0.5px;  
    border-style:solid;
    border-color: rgb(51, 153, 255);  
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left:15px;      
    margin-left: 5px;
    margin-right: 5px;
    padding:1px;
}</string>
     </property>
     <property name="title">
      <string>EKS钥匙设置</string>
     </property>
     <widget class="QGroupBox" name="groupBo_wirte">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>411</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>写:</string>
      </property>
      <widget class="QPushButton" name="pushButton_green">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>40</y>
         <width>90</width>
         <height>90</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style: solid;
background-color: rgb(8, 129, 60);</string>
       </property>
       <property name="text">
        <string>绿</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_black">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>42</y>
         <width>90</width>
         <height>90</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style: solid;
background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>黑</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_red">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>90</width>
         <height>90</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);

border-style: solid;
background-color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>红</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_key_write">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>131</width>
         <height>35</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(250, 0, 0);</string>
       </property>
       <property name="text">
        <string>写入成功!</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_read">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>90</y>
        <width>90</width>
        <height>90</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/35_all/67.bmp);
font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);</string>
      </property>
      <property name="text">
       <string>读</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_key_read">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>210</y>
        <width>131</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(250, 0, 0);</string>
      </property>
      <property name="text">
       <string>写入成功!</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>181</width>
       <height>571</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget{ border-width:1px;  
					border-style:solid;		
					border-color: rgb(51, 153, 255); }

QListWidget::item:selected{background: rgb(51, 153, 255); color:white}
QListWidget::item:selected:!active{border-width:0px; background: rgb(51, 153, 255); color:white}</string>
     </property>
     <item>
      <property name="text">
       <string>PHEVENCODER</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PHEVIOPOSITION</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BEVENCODER</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BEVIOPOSITION</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </item>
    </widget>
    <widget class="QTableWidget" name="tableEncoder">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>801</width>
       <height>571</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
    font: 14pt &quot;黑体&quot;; 
	border-width:1px;  
	border-style:solid;
	border-color: rgb(51, 153, 255);  
	background-color: rgb(248, 248, 255);
}
QTableWidget::Item{

}
QTableWidget::Item:hover{

}
QTableWidget::item:selected{
	background:lightgray; 
	color:rgb(248, 248, 255); 
}
QTableWidget::item:selected:!active{

}
QTableCornerButton::section{
	background-color: rgb(51, 153, 255);
}
QScrollBar:vertical{
width:36px;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAdd">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>700</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style: solid;
background-color: rgb(51,153,255);</string>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnInsert">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>700</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style: solid;
background-color: rgb(51,153,255);</string>
     </property>
     <property name="text">
      <string>插入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEdit">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>700</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style: solid;
background-color: rgb(51,153,255);</string>
     </property>
     <property name="text">
      <string>编辑</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDelete">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>700</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style: solid;
background-color: rgb(51,153,255);</string>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSave">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>700</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style: solid;
background-color: rgb(51,153,255);</string>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>273</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioBtnEn">
        <property name="styleSheet">
         <string notr="true">QRadioButton{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QRadioButton::indicator { 
   width: 25px;
   height: 25px;
   image: url(:/LCD_CS351/LCD_CS351/logo/Radio.png);
}

QRadioButton::indicator:checked { 
image: url(:/LCD_CS351/LCD_CS351/logo/Radio_check.png);
}</string>
        </property>
        <property name="text">
         <string>ENCODER</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioBtnIO">
        <property name="styleSheet">
         <string notr="true">QRadioButton{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QRadioButton::indicator { 
   width: 25px;
   height: 25px;
   image: url(:/LCD_CS351/LCD_CS351/logo/Radio.png);
}

QRadioButton::indicator:checked { 
image: url(:/LCD_CS351/LCD_CS351/logo/Radio_check.png);
}</string>
        </property>
        <property name="text">
         <string>IO</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btnAddList">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>646</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style: solid;
background-color: rgb(51,153,255);</string>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDeleteList">
     <property name="geometry">
      <rect>
       <x>151</x>
       <y>646</y>
       <width>90</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style: solid;
background-color: rgb(51,153,255);</string>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QLabel" name="shageLabel">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>760</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
background-color: rgb(51, 153, 255);




</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>40</y>
      <width>57</width>
      <height>57</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image: url(:/re/back.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_rfid">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>167</y>
      <width>17</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/logo/triangle.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rfid">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>160</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    border:none;
    background:transparent;
    font: 18pt &quot;黑体&quot;;
    color: rgb(248, 248, 255);
}</string>
    </property>
    <property name="text">
     <string>RFID/EKS</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEncoder">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    border:none;
    background:transparent;
    font: 18pt &quot;黑体&quot;;
    color: rgb(248, 248, 255);
}</string>
    </property>
    <property name="text">
     <string>ENCODER</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
